frontend/server/src/ProblemParams.php

```diff
@@ -334,4 +334,33 @@ public function updateValueParams(
         }
         return $importantChange;
     }
+
+    /**
+    * Convert string visibility to numeric visibility
+    */
+    public static function stringVisibilityToNumeric(string $visibility): int {
+        switch ($visibility) {
+            case 'deleted':
+                return \OmegaUp\ProblemParams::VISIBILITY_DELETED;
+            case 'private_banned':
+                return \OmegaUp\ProblemParams::VISIBILITY_PRIVATE_BANNED;
+            case 'public_banned':
+                return \OmegaUp\ProblemParams::VISIBILITY_PUBLIC_BANNED;
+            case 'private_warning':
+                return \OmegaUp\ProblemParams::VISIBILITY_PRIVATE_WARNING;
+            case 'private':
+                return \OmegaUp\ProblemParams::VISIBILITY_PRIVATE;
+            case 'public_warning':
+                return \OmegaUp\ProblemParams::VISIBILITY_PUBLIC_WARNING;
+            case 'public':
+                return \OmegaUp\ProblemParams::VISIBILITY_PUBLIC;
+            case 'promoted':
+                return \OmegaUp\ProblemParams::VISIBILITY_PROMOTED;
+            default:
+                throw new \OmegaUp\Exceptions\InvalidParameterException(
+                    'invalidVisibility',
```

Comment:
puedes agregar esta cadena de traducción o cambiarlo a 

```suggestion
                    'parameterInvalid',
```

?

Reply:
Voy agregar la cadena de traducción

Reply:
@tvanessa @lhchavez me podrías ayudar con el texto para la cadena de traducción se me ocurre algo como "La visibilidad del problema no esta dentro de los valores permitidos" o "El %(valor) no esta dentro de los valores permitidos para la visibilidad"

Reply:
Puede ser algo que indique cuáles son los valores permitidos, como "El parámetro `visibility` puede ser alguno de los siguientes valores: (lista de valores)"

Reply:
si gracias